src/content/en/2019/markup.md

```diff
@@ -0,0 +1,301 @@
+
+And [`<app-root>` (generally a framework like Angular) appears on 8.2k mobile sites](https://rainy-periwinkle.glitch.me/permalink/6997d689f56fe77e5ce345cfb570adbd42d802393f4cc175a1b974833a0e3cb5.html) ([8.5k on desktop](https://rainy-periwinkle.glitch.me/permalink/ee3c9dfbcab568e97c7318d9795b9ecbde0605f247b19b68793afc837796aa5c.html)).
+
+Comparing these to a few of the native HTML elements that are below the 5% bar, for perspective, looks something like this (note - varies slightly based on dataset).
+
+(( TOOD: there is a corresponding image in the google doc https://docs.google.com/document/d/16TY_pV-FyW35DzuvdlaOiENz6o6PWYpl_RviU-HW7Qc/edit ))
+
+
+You could draw interesting observations like these all day long.
+
+Here's one that's a little different:  Productization causing popularity is evident in outright errors as well. [`<pclass="ddc-font-size-large"`](https://rainy-periwinkle.glitch.me/permalink/3214f840b6ae3ef1074291f60fa1be4b9d9df401fe0190bfaff4bb078c8614a5.html)[ > was a parsed tag name which occurred in our dataset in over 1000 sites](https://rainy-periwinkle.glitch.me/permalink/3214f840b6ae3ef1074291f60fa1be4b9d9df401fe0190bfaff4bb078c8614a5.html)  This was thanks to a missing space in a popular 'as a service' kind of product.  Happily, we reported this error during our research and it was quickly fixed.
+
+In his original paper, Hixie mentions that "The good thing, if we can be forgiven for trying to remain optimistic in the face of all this non-standard markup, is that at least these elements are all clearly using vendor-specific names. This massively reduces the likelihood that standards bodies will invent elements and attributes that clash with any of them."  However, as mentioned above, this is not universal.  Over 25% of the non-standard elements that we captured don't use any kind of namespacing strategy to avoid polluting the global namespace.  Here is [a list of 1157 elements like that from the mobile dataset](https://rainy-periwinkle.glitch.me/permalink/53567ec94b328de965eb821010b8b5935b0e0ba316e833267dc04f1fb3b53bd5.html).  Many of those, as you can see, are probably non-problematic as they are obscure names, misspellings and so on - but at least a few probably present some challenges.  You'll note, for example,  that `<toast>` (which Googlers [recently tried to propose as `<std-toast>`)](https://www.chromestatus.com/feature/5674896879255552) appears in this list.
```

Comment:
```suggestion
In his original paper, Hixie mentions that "The good thing, if we can be forgiven for trying to remain optimistic in the face of all this non-standard markup, is that at least these elements are all clearly using vendor-specific names. This massively reduces the likelihood that standards bodies will invent elements and attributes that clash with any of them."  However, as mentioned above, this is not universal.  Over 25% of the non-standard elements that we captured don't use any kind of namespacing strategy to avoid polluting the global namespace.  Here is [a list of 1157 elements like that from the mobile dataset](https://rainy-periwinkle.glitch.me/permalink/53567ec94b328de965eb821010b8b5935b0e0ba316e833267dc04f1fb3b53bd5.html).  Many of those, as you can see, are probably non-problematic as they are obscure names, misspellings and so on -- but at least a few probably present some challenges.  You'll note, for example,  that `<toast>` (which Googlers [recently tried to propose as `<std-toast>`](https://www.chromestatus.com/feature/5674896879255552)) appears in this list.
```